### DISCUSSION TITLE
Does MONAI/PyTorch work in Windows Subsystem for Linux [WSL2]?

### DISCUSSION THREAD

**Discussion by dzenanz** [Project-MONAI/MONAI#7221] [General]
Auto3DSeg currently does not work on Windows, but does it work on WSL2? My Linux box is [too weak](https://github.com/Project-MONAI/MONAI/discussions/7214) to be useful for Auto3DSeg.

**Comment by KumoLiu** [comment]
Hi @dzenanz, It shouldn't be an issue in theory because wsl is a complete Linux system. But we probably didn't test it in a wsl environment either.

Could you please help double-confirm it? @mingxin-zheng Thanks!
